frontend/src/utils/actions.ts

```diff
@@ -2,6 +2,54 @@
 
 import { profileSchema } from "./schemas";
 
+const MockProperty = [
```

Comment:
convention in many languages is to have constant names in uppercase with underscores between words. class names are usually Pascal/title case.

Reply:
Javascript uses camelcase for constant. I will change it. Thank you for pointing it out.